lib/features/dataset/display.dart

```diff
@@ -34,9 +35,12 @@ import 'package:flutter_riverpod/flutter_riverpod.dart';
 import 'package:rattle/constants/app.dart';
 import 'package:rattle/constants/markdown.dart';
 import 'package:rattle/constants/spacing.dart';
+import 'package:rattle/features/dataset/config.dart';
 import 'package:rattle/providers/meta_data.dart';
 import 'package:rattle/providers/page_controller.dart';
 import 'package:rattle/providers/path.dart';
+import 'package:rattle/providers/rolesTableRebuild.dart';
```

Comment:
This is NOT the style of file naming we are using EVERY WHERE ELSE in this code - why have you suddenly changed the style?

Reply:
done